hazelcast-client/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -970,7 +970,7 @@ private void handStats(String[] args) {
         }
     }
 
-    @SuppressWarnings("LockAcquiredButNotSafelyReleased")
+    @SuppressWarnings({"LockAcquiredButNotSafelyReleased", "squid:S2222"})
```

Comment:
@lazerion Could you just please explain this syntax to me?

Reply:
@Holmistr `squid:S2222` is sonar rule id to suppress check. I will also double check after we got new analysis.

Reply:
@kwart 
Link to [rule](https://sonarcloud.io/organizations/devopshazelcast-github/rules?open=squid%3AS2222&rule_key=squid%3AS2222)
Link to reference [solution](https://stackoverflow.com/a/29400824/305038)

Reply:
I mean add it directly to the source. Sth. like
```java
// squid:S2222 suppression avoids false positive finding in SonarQube scans ... blah blah...
@SuppressWarnings({"LockAcquiredButNotSafelyReleased", "squid:S2222"})
```

Reply:
@lazerion I tend to agree with @kwart , with that comment it will be obvious from the code what it means, so nobody will ever forget.

Reply:
@Holmistr @kwart got the comment, I will add comment to code, not issue :) thanks